### DISCUSSION TITLE
Request for feedback: New “Connections” network graph in DataCite Commons

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#130] [Requests for Comment]
## A new visualization tool in DataCite Commons

DataCite is introducing a new “Connections” network graph in [DataCite Commons](https://commons.datacite.org/), and **we’d love to hear your feedback** on this new visualization tool! The graph is now available on all Works pages.

To highlight the diversity of research outputs and their interconnectedness, the force-directed network graph uses DOI [related identifier](https://support.datacite.org/docs/connecting-to-works#related-identifiers) metadata to visualize the connections among resource types connected to a DOI. Works and connections are grouped by resource type to allow for a broader visualization of the connections within the DataCite data file.

<img width="728" alt="Screen Shot 2024-10-21 at 5 45 14 PM" src="https://github.com/user-attachments/assets/71bdb9fa-8199-47c9-96cf-83670e7afa37">

Start exploring the “Connections” network graph by:

- Visiting [an example Work page for a Project DOI in Commons](https://commons.datacite.org/doi.org/10.60581/zaev-6p15) 
- Consulting [our Support documentation](https://support.datacite.org/docs/works-in-datacite-commons#network-graph) about the graph

## Providing feedback

We’re looking into future enhancements to the network graph and we’re eager to hear how we can best support your needs and use cases. **Contribute your feedback** using [the suggestion form](https://github.com/datacite/datacite-suggestions/discussions/new?category=general-suggestions) or by leaving a comment on this post. You’re welcome to reply by answering any of the questions below: 

- How are you using the network graph? 
- What would you change about the graph as it exists now? 
- When you visit a Work page in Commons, what information, visualizations, or reports do you hope to see that aren’t there yet?

_This discussion will close on 13 December 2024, but you can still submit feedback on [DataCite Suggestions](https://github.com/datacite/datacite-suggestions/discussions/67)._

**Comment by adambuttrick** [comment]
Some feedback:

- The Connections graph describes relationships between objects, but there is a nearby UI element "Connection Types" for filtering to relationships of a different meaning/kind. Ideally, these would be better disambiguated, especially as they have some overlapping categories (i.e. Other)

- Perhaps separate, toggable graphs for connected objects vs. connection types, but even here there is opportunity for combine. For example, I could have a graph of connected types and when I click into say references, it reveals all related object types that reference (project, journal articles, etc).

- The overall absence of interactivity in the graph. For example, when clicking on "Journal Article," I would expect some other UI action to follow (e.g. a filter to be applied to the item list below, returning these items,) vs. simply being able to drag around.

**Reply by codycooperross** [reply]
Thanks, @adambuttrick —this is really helpful. I hear that there could be more clarity and a stronger representation of connection types in addition to connected objects. Interactivity could certainly help here. What are some ways that you're using the graph or hoping to use the graph?

**Reply by adambuttrick** [reply]
@codycooperross We use DataCite commons/this graph when we're demoing the value of ROR for institutional identification of datasets and data citation practices more broadly. Most recently, for some US federal government funding agencies.
